.circle/pack.yaml.schema

```diff
@@ -8,6 +8,7 @@ properties:
     type: array
   version:
     type: string
+    pattern: "^(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)\.(?:0|[1-9]\d*)(?:-[\da-z\-]+(?:\.[\da-z\-]+)*)?(?:\+[\da-z\-]+(?:\.[\da-z\-]+)*)?$"
```

Comment:
It wouldn't be me if there would be no regexes involved! 😛 Added the description.